src/main/java/com/datapipe/jenkins/vault/VaultAccessor.java

```diff
@@ -20,22 +20,26 @@
 
 
     public void init(String url) {
-        init(url, null, false);
+        init(url, null, false, null);
     }
 
     public void init(String url, boolean skipSslVerification) {
-        init(url, null, skipSslVerification);
+        init(url, null, skipSslVerification, null);
     }
 
     public void init(String url, VaultCredential credential) {
-        init(url, credential, false);
+        init(url, credential, false, null);
     }
 
     public void init(String url, VaultCredential credential, boolean skipSslVerification) {
+        init(url, credential, skipSslVerification, null);
+    }
+
+    public void init(String url, VaultCredential credential, boolean skipSslVerification, String vaultCACert) {
```

Comment:
Can we switch to a builder pattern this is getting crazy 😆 
I know #34 is also mocking about there
Actually, I like #34 approach more using a VaultConfig

Reply:
So, do you want to merge #34 first? Would prefer not have to keep refactoring things.

Reply:
#34 is not ready for merge, I asked for changes :swe

Reply:
Also the PR have conflicts, thanks to me trying to fixup the project, so sorry 😓